### PR TITLE
chore(QTM-283): Converts regression tests stories to CSF

### PR DESCRIPTION
**Pull request by toncabral88**
## Description
https://jirasoftware.catho.com.br/browse/QTM-283

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### CODE REVIEW

File: stories/Input/Input.story.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default {
   title: 'Forms',
 };
 
-export const _Input = () => (
+export const InputStory = () => (
```
Comment (rapahaeru) [inline stories/Input/Input.story.jsx:29]: The description text is missing here:

how about:

```
const description="The Input component is used to create interactive controls for web-based forms in order to accept data from the user."

export const InputStory = () => (
  <>
    <AutoExample
      description={description}
      component={Input}
      componentProps={{
        id: 'example',
        name: 'example',
        label: 'Example',
      }}
      additionalTabs={tabExample}
    />
  </>
```

**Review by rapahaeru — APPROVED** [review]
## Review guide
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [x] Code review

**Review by rapahaeru — DISMISSED** [review]

File: stories/RadioGroup/RadioGroup.regression-test.story.jsx [hunk 2]
```diff
@@ -17,35 +17,34 @@ const options = [
   },
 ];
 
-const disabledOptions = [...options];
-disabledOptions[1] = {
-  value: 'Mustard',
-  label: 'Mustard',
-  disabled: true,
-};
+const disabledOptions = [
```
Comment (matheusrpp) [inline stories/RadioGroup/RadioGroup.regression-test.story.jsx:20]: Question. Have you noticed that this guy is adding another option and not just chaging the one it already have?

**Review by matheusrpp — APPROVED** [review]
Just pointed two things, but nothing that blocks this pr. 
## Review guide
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [x] Code review

File: stories/RadioGroup/RadioGroup.regression-test.story.jsx (hunk 2 above)
Comment (toncabral88) [reply stories/RadioGroup/RadioGroup.regression-test.story.jsx:20]: Yep! It was already similar with this.

File: stories/Accordion/Accordion.regression-test.story.jsx [hunk 3]
```diff
@@ -9,34 +9,19 @@ import { Accordion } from '../../components';
...
-AccordionWithFirstItemOpened.story = {
-  name: 'Accordion with first item opened',
-};
+export const FirstItemOpened = Template.bind({});
+FirstItemOpened.args = { ...mockWithFirstItemOpened };
 
-export const AccordionWithLastItemOpened = () => (
-  <Accordion {...mockWithLastItemOpened} />
-);
+export const LastItemOpened = () => <Accordion {...mockWithLastItemOpened} />;
```
[the remark this answers is not in the document]
Comment (toncabral88) [reply stories/Accordion/Accordion.regression-test.story.jsx:23]: Wow! I don't saw this.

**Review by rapahaeru — APPROVED** [review]

**Review by matheusrpp — APPROVED** [review]

**Pull request merged**